app/src/main/java/org/coepi/android/ui/symptoms/cough/CoughDurationFragment.kt

```diff
@@ -29,4 +30,11 @@ class CoughDurationFragment : Fragment() {
         onBack { viewModel.onBack() }
         toolbar.setNavigationOnClickListener { viewModel.onBackPressed() }
     }.root
+
+    override fun onStop() {
+        val imm: InputMethodManager =
```

Comment:
Put this in a class. For now you can instantiate it here, e.g. `KeyboardHider().hide()`